templates/terraform/staging-storage/main.tf

```diff
@@ -69,15 +69,17 @@ module staging_notification_pubsub_topic {
 }
 
 resource google_pubsub_subscription_iam_member staging_writer_iam {
+  depends_on   = [module.staging_notification_pubsub_topic]
```

Comment:
So the "depends_on" argument should be used when there's a dependency on that module/resource but it isn't explicitly stated or referenced within the other arguments?

Reply:
The `depends_on` is often used in the very case Drew is facing; we don't want to create a particular resource _until_ the resource is actually created. Here, we don't want to create the writer iam for the pubsub topic until the topic actually exists, or else we will quite possibly run into an error (something like "Can't create the IAM for this topic because the topic doesn't exist")